.github/workflows/terminate-zombie-build-instances.yml

```diff
@@ -34,3 +34,9 @@ jobs:
 
           # See https://docs.aws.amazon.com/cli/latest/reference/ec2/terminate-instances.html for terminate command.
           echo $to_terminate |  jq '.[]  | .InstanceId' | xargs --no-run-if-empty --max-args=1 aws ec2 terminate-instances --instance-ids
+
+    steps:
+      - shell: List and Terminate GH actions runners Older than One Day
```

Comment:
I don't see logic to filter out runners older than a day. Am I missing something?

Reply:
that may have been an aspirational day when I started writing the script, I'll update the comment.  

We are deleting anything not "online" but I'm changing that in an upcoming PR to `==` offline which seems to be the end state of these runners.  Happy to add day logic though

Reply:
no need. wanted to make sure I understood the logic. I think filtering for offline makes sense. I believe the action status is online -> offline. would double check that runners don't start in offline.